src/elements/common/modal.scss

```diff
@@ -6,6 +8,10 @@
         justify-content: center;
         padding: 15px 0 0;
 
+        [class^='bp_base_button_module_button']:last-child {
```

Comment:
have to put this here until we can use `button` instead once all the modals have been converted